applications/PfemFluidDynamicsApplication/custom_processes/generate_new_nodes_before_meshing_process.hpp

```diff
@@ -924,12 +940,12 @@ namespace Kratos
 										if (diffX < 0 && diffY < 0) // the node is in the same zone of a previously inserted node
 										{
 											// std::cout << " the nodes has more or less the same position of a previously inserted node" << NewPositions[j][0] << " " << NewPositions[j][1] << " versus " << NewPosition[0] << " " << NewPosition[1] << std::endl;
-											suitableElement = false;
+											suitableElementForSecondAdd = false;
 										}
 									}
 								}
 
-								if (suitableElement == true)
+								if (suitableElementForSecondAdd == true)
```

Comment:
you can use `if (suitableElementForSecondAdd ) {`

Reply:
done!